app/lib/clients/twitter_client.rb

```diff
@@ -11,11 +11,36 @@ def self.authenticate
     client
   end
 
+  TwitterQuery = Struct.new(:created_at, :text, :favorite_count, :retweet_count, :reply_count)
+
+  def self.reply_count(id, client)
+    replies = client.search("to:mojotech", since_id: id, include_entities: true)
+                    .select { |res| res.in_reply_to_status_id == id }
+    replies.count
+  end
+
+  def self.remove_hyperlinks(text)
+    links = text.split(/\s|\n/).select { |word| word.include? "http" }
+    clean_text = text
+    links.each do |l|
+      clean_text = clean_text.gsub(l, "")
+    end
+    clean_text
+  end
+
   def self.latest_tweet
     client = authenticate
     # API gets count amt, then filters out rts and replies
     # This guarantees there will always be a tweet
-    tweet = client.user_timeline('mojotech', count: 5, include_rts: false, exclude_replies: true).first
-    tweet
+    latest = client
```

Comment:
I might be missing something, but I think you could remove that initial commit entirely -- having this code change makes it a little harder for me to get a model of what it's doing. (If I missed something, pls lemme know!)